dashboard/app/models/level_source_image.rb

```diff
@@ -21,6 +21,9 @@
 class LevelSourceImage < ActiveRecord::Base
   belongs_to :level_source
 
+  S3_BUCKET = 'cdo-art'.freeze
+  S3_URL = 'https://d3p74s6bwmy6t9.cloudfront.net/'.freeze
```

Comment:
No idea. No idea. The diff in this PR is simply my moving it to the top of the file.